src/v3/test/integration/__snapshots__/okta-verify-enrollment-version-upgrade.test.tsx.snap

```diff
@@ -199,7 +199,7 @@ exports[`okta-verify-enrollment-version-upgrade should render form 1`] = `
                           <li
                             class="MuiListItem-root emotion-33"
                           >
-                            On your mobile device, download the Okta Verify app from the App Store (iPhone and iPad) or Google Play (Android devices).
+                            On your other device, download the Okta Verify app from the App Store® (iPhone® and iPad®) or on Google Play (Android? devices).
```

Comment:
Might want to check the translation string here - "Android" has a `?` showing here

Reply:
looks like this key was originally added here: https://github.com/okta/okta-signin-widget/commit/df224e3be2611c88f9816301b02c049161be84ef#:~:text=oie.enroll.okta_verify.qrcode.step1.updated%20%3D%20On%20your%20other%20device%2C%20download%20the%20Okta%20Verify%20app%20from%20the%20App%20Store%C2%AE%20(iPhone%C2%AE%20and%20iPad%C2%AE)%20or%20on%20Google%20Play%20(Android%3F%20devices).

with a "?" by mistake.  Asking globalization if this can be fixed manually